### ISSUE TITLE
switching tabs when batch uploading is broken

### ISSUE THREAD

**Issue by ShadowVoid** [genomizer/genomizer-desktop#67]
was uploading 200 files, switched to another tab, switched back, worked fine
did it again extremely fast and repeatedly,  now all the bars are stuck at 0%

**Comment by ShadowVoid** [comment]
no, it's just the case that if you switch tabs, the current upload finishes, but the next one doesn't start.

**Comment by rednatto** [comment]
What is the issue? Needs clarification.
